### DISCUSSION TITLE
File Tags

### DISCUSSION THREAD

**Discussion by LeStarch** [nasa/fprime#1896] [Ideas]
File tags are a way of capturing metadata about files in the filesystem.  How should F´  support this concept.

**Comment by SterlingPeet** [comment]
I would tend to think that file metadata exploration should probably be its own component.  The ground operator could issue a few commands to look through the files, and then select one to downlink via the FileDownlink component, so long as the filesystem reference handle (filename in this case, I think) is the same.
